src/interfaces/Client/IClientJobs.ts

```diff
@@ -11,17 +11,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { JobSchedule } from "../../types/jobs/JobSchedule.type";
+import { Schedule } from "../../types/jobs/JobSchedule.type";
```

Comment:
Especially if the file only contains a single type, the type name should match the file name.